src/Item/Data.elm

```diff
@@ -190,26 +300,70 @@ type alias Coins =
     }
 
 
-type alias Purse =
-    { base : BaseItem
-    , coins : Coins
+type alias Purse compatible =
+    { compatible
+        | name : String
+        , prices : Prices
+        , css : String
+        , mass : Mass
+        , status : ItemStatus
+        , isIdentified : IdentificationStatus
+        , coins : Coins
     }
 
 
-type alias CopperCoins =
-    { base : BaseItem, value : Int }
+type alias CopperCoins compatible =
+    { compatible
+        | name : String
+        , prices : Prices
+        , css : String
+        , mass : Mass
+        , status : ItemStatus
+        , isIdentified : IdentificationStatus
+        , value : Int
+    }
+
 
+type alias SilverCoins compatible =
+    { compatible
+        | name : String
+        , prices : Prices
+        , css : String
+        , mass : Mass
+        , status : ItemStatus
+        , isIdentified : IdentificationStatus
+        , value : Int
+    }
 
-type alias SilverCoins =
-    { base : BaseItem, value : Int }
 
+type alias GoldCoins compatible =
+    { compatible
+        | name : String
+        , prices : Prices
+        , css : String
+        , mass : Mass
+        , status : ItemStatus
+        , isIdentified : IdentificationStatus
+        , value : Int
+    }
 
-type alias GoldCoins =
-    { base : BaseItem, value : Int }
 
+type alias PlatinumCoins compatible =
+    { compatible
+        | name : String
+        , prices : Prices
+        , css : String
+        , mass : Mass
+        , status : ItemStatus
+        , isIdentified : IdentificationStatus
+        , value : Int
+    }
 
-type alias PlatinumCoins =
-    { base : BaseItem, value : Int }
+
+type BeltContainer a
+    = TwoSlot ( Maybe a, Maybe a )
+    | ThreeSlot ( Maybe a, Maybe a, Maybe a )
+    | FourSlot ( Maybe a, Maybe a, Maybe a, Maybe a )
```

Comment:
Curious as to why you're using tuples here. Doesn't that add another layer of structure where you maybe don't need one?  ie `TwoSlot (Maybe a) (Maybe a)` rather than `TwoSlot (Maybe a, Maybe a)` ?